.github/workflows/betanet_v1.1.yml

```
@@ -41,4 +41,6 @@ jobs:
 
     - name: Run tests
       working-directory: contracts
-      run: npm install ; npm run test:gw_betanet_v1
+      run: |
```

```suggestion
      run: |
      if: ${{ secrets.GODWOKEN_BETANET_PRIVATE_KEY != '' }}
```

Fork repos may not set this secret. Skip them.

---

> Fork repos may not set this secret. Skip them.

Should I just add the condition to the job, so if fork repo has no serect set, then skip the whole test.
```yml
jobs:
  contract-tests:
    if: ${{ secrets.GODWOKEN_TESTNET_PRIVATE_KEY != '' }} && ${{ secrets.GODWOKEN_TESTNET_PRIVATE_KEY2 != '' }}
```

---

Codes in the last comment was a wrong attempt.
I have commited some new changes about the `secret checking` function, now it should work.